_data/sites.json

```
@@ -440,6 +440,15 @@
         "notes_it-IT": "L'unico modo per contattarli è tramite email, ma nessuno risponde."
     },
 
+    {
+        "name": "Modrinth",
+        "url": "https://modrinth.com/legal/privacy",
+        "difficulty": "hard",
```

Since the emails is `gdpr@modrinth.com` I guess you need to live in a country where the GDPR applies, right?
Because that would be difficulty `limited`, instead of `hard`.

```suggestion
        "difficulty": "limited",
```

---

From their privacy policy:
> ## Access, rectification, erasure, restriction, portability, and objection
> Every user is entitled to the following:
> 
> The right to access – You have the right to request copies of your personal data. We may charge you a small fee for this service.
> The right to rectification – You have the right to request that we correct any information you believe is inaccurate. You also have the right to request us to complete the information you believe is incomplete.
> The right to erasure – You have the right to request that we erase your personal data, under certain conditions.
> The right to restrict processing – You have the right to request that we restrict the processing of your personal data, under certain conditions.
> The right to data portability – You have the right to request that we transfer the data that we have collected to another organization, or directly to you, under certain conditions.
> The right to object to processing – You have the right to object to our processing of your personal data, under certain conditions.
> If you would like to exercise those rights, contact us at [gdpr@modrinth.com](mailto:gdpr@modrinth.com). We may ask you to verify your identity before proceeding and will respond to your request within 30 days as required by law, or notify you of an extended reply time.

Anyway I will check if everyone can access their data.

---

@Kefaku they replied and sent me the data without verifying my country.